Mathlib/Analysis/SpecialFunctions/Log/Basic.lean

```diff
@@ -484,40 +481,47 @@ end TendstoCompAddSub
 namespace Mathlib.Meta.Positivity
 open Lean.Meta Qq
 
-lemma log_nonneg_of_isNat (h : NormNum.IsNat e n) : 0 ≤ Real.log (e : ℝ) := by
+variable {e : ℝ} {d : ℕ}
```

Comment:
maybe add `n` to this one, to avoid it in the next three lemmas?

Reply:
I tried this! But it interferes with the four lemmas still using autoImplicit: each proof by `exact_mod_cast of_decide_eq_true h` errors with an error 

```
mod_cast has type
  @OfNat.ofNat ℚ 1
      (@One.toOfNat1 ℚ
        (@AddMonoidWithOne.toOne ℚ
          (@AddGroupWithOne.toAddMonoidWithOne ℚ (@Ring.toAddGroupWithOne ℚ NormedRing.toRing)))) <
    ↑n / ↑d : Prop
but is expected to have type
  @OfNat.ofNat ℝ 1
      (@One.toOfNat1 ℝ
        (@AddMonoidWithOne.toOne ℝ
          (@AddGroupWithOne.toAddMonoidWithOne ℝ (@Ring.toAddGroupWithOne ℝ Real.instRingReal)))) <
    ↑n / ↑d : Prop.
```
I don't know off-hand how to fix that. (It would be a welcome follow-up cleanup!)